packages/truffle-box/lib/utils/index.js

```diff
@@ -29,34 +29,26 @@ module.exports = {
       return path || (fs.existsSync(alt) && alt);
     }, undefined);
 
-    try {
-      const boxConfig = await config.read(configPath);
-      return boxConfig;
-    } catch (error) {
-      throw new Error(error);
-    }
+    return await config.read(configPath);
   },
 
-  setUpTempDirectory: () => {
-    const prepareSpinner = ora("Preparing to download").start();
-    return new Promise((resolve, reject) => {
-      const options = {
-        dir: cwd,
-        unsafeCleanup: true
+  setUpTempDirectory: async () => {
```

Comment:
Check me on this, but you should make this method just plain old synchronous. It gets called from `truffle-box/box` and it looks like you can just remove the await from that file. Simplification!

Reply:
Ah, looks like it can be sync, but still need to keep the `await` in `box.js` within the `unbox` method.